doc/api/url.md

```diff
@@ -525,6 +525,45 @@ console.log(JSON.stringify(myURLs));
 // Prints ["https://www.example.com/","https://test.example.org/"]
 ```
 
+#### Static: URL.from(input)
```

Comment:
Other docs usually use "Class Method: " instead of "Static: ". Also, these methods are mostly placed before the instance methods. See, for example, TOC in the [`Buffer` doc](https://nodejs.org/api/buffer.html).

Reply:
Will fix it.

Reply:
```suggestion
#### Class Method: URL.from(input)
```